tokio/src/fs/open_options.rs

```diff
@@ -382,6 +382,13 @@ impl OpenOptions {
         let std = asyncify(move || opts.open(path)).await?;
         Ok(File::from_std(std))
     }
+
+    /// Returns an immutable reference to the the underlying std::fs::OpenOptions
+    #[doc(hidden)]
```

Comment:
The `#[doc(hidden)]` attribute is not necessary on private methods.

Reply:
will remove, thanks